asciidoctor_installation.adoc

```diff
@@ -0,0 +1,100 @@
+= Introduction to Asciidoctor
+Asciidoctor is a fast text processor and publishing toolchain for converting AsciiDoc content to HTML5, DocBook and other formats. It is written in Ruby and packaged as a Ruby gem.
+
+== Installing Asciidoctor
+
+.Prerequisites
+* Asciidoctor requires one of the following implementations of Ruby:
+** Ruby version 2.3 or later
+** JRuby version 9.1 or later
+** TruffleRuby version 20.2 or later
+* Supported operating systems:
+** Ubuntu versions18.04 LTS and 20.04 LTS
+** Fedora version 33 or later
+** macOS Mojave, Catalina, and Big Sur
+** Microsoft Windows 10 and Windows Server 2019
+
+.Procedure
```

Comment:
I suggest we also say to install https://pkgs.org/search/?q=asciidoctor-pdf so folks can build PDFs and not just HTML.